variable-replacers/README.md

```diff
@@ -0,0 +1,93 @@
+# Pluggable Variable Replacers
+
+This code sample shows how to implement and use the pluggable configuration replacer feature introduced in Hazelcast
+3.10.
+
+## Build and run the samples
+
+Build this sample with Maven.
+```
+mvn clean install
+```
+
+There are several Maven profiles prepared to execute the samples in the `pom.xml`. Use following command to run them:
+
+```bash
+mvn exec:java -P [profileName]
+```
+
+| Profile name | Related XML file | Description |
+| ------------ | ---------------- | ----------- |
+| `show.encryption-help` | | Executes the EncryptionReplacer without parameters. It just prints a help. | 
+| `generate.encrypted-variable` | `hazelcast-with-replacers.xml` | Executes the EncryptionReplacer with arguments provided. It encrypts the provided String with configuration loadad from the XML file. The resulting Variable is printed to standard output. |
+| `run.hazelcast-plain` | `hazelcast-plain.xml` | Starts Hazelcast server without replacers used. |
+| `run.hazelcast-with-replacers` | `hazelcast-with-replacers.xml` | Starts Hazelcast server with `EncryptionReplacer` and sample `IdReplacer` used in the configuration. |
+| `run.hazelcast-exec-linux` | `hazelcast-exec-linux.xml` | Starts Hazelcast server with sample `ExecReplacer` used in the configuration. The command used (`echo`) should be available on most Unix/Linux-like systems. |
+| `run.hazelcast-exec-windows` | `hazelcast-exec-windows.xml` | Starts Hazelcast server with sample `ExecReplacer` used in the configuration. The commands used (`cmd.exe` and its `echo`) should be available on most Unix/Linux-like systems. |
```

Comment:
> should be available on most Unix/Linux-like systems

Should be Windows.

Reply:
Nice catch! The C/P is no longer my friend when it does such mistakes. I'll fix it.